op-node/flags/flags.go

```diff
@@ -148,7 +149,7 @@ var (
 	MetricsAddrFlag = &cli.StringFlag{
 		Name:    "metrics.addr",
 		Usage:   "Metrics listening address",
-		Value:   "0.0.0.0",
+		Value:   "127.0.0.1",
```

Comment:
I like this change, but it's a breaking change so we'll need to be careful with it and make sure it's very clearly communicated. Probably worth considering flagging it as an upcoming breaking change for a release or two before actually changing it so people can specify it explicitly if they do want it to be publicly available.

Reply:
We still seem to have the breaking change to defaults in this PR though. Suspect we need to go back to 0.0.0.0 as the default so we can get this merged.

Reply:
nice catch, I forgot to revert the change for these flags